requirements/edx-sandbox/base.in

```diff
@@ -16,14 +15,15 @@
 -c ../constraints.txt
 
 -r shared.txt                       # Dependencies in common with LMS and Studio
-chem                                # A helper library for chemistry calculations
+chem
 matplotlib==2.2.4                   # 2D plotting library
+networkx==2.2                       # Utilities for creating, manipulating, and studying network graphs
 numpy==1.16.5                       # Numeric array processing utilities; used by scipy
+openedx-calc
 pyparsing==2.2.0                    # Python Parsing module
-random2==1.0.1                      # Implementation of random module that works identically under Python 2 and 3
+random2                             # Implementation of random module that works identically under Python 2 and 3
```

Comment:
This loses its pin -- intentional? (and openedx-calc is not pinned, above)

Reply:
We did specifically pin random2 recently, and probably want openedx-calc pinned also.  The thinking is that unlike the main LMS requirements, we want the codejail ones to stay static until we're explicitly ready to resolve any potential breakage that upgrades trigger in course team code.

Reply:
Yea, that's intentional, I think we want to maintain the latest version for those libraries since for calc, it's install in the sandbox and in the LMS we want those to be in sync.  I'm not expecting much updates to rando2 other than updates to make it continue to function with newer versions of python which we'll want to pull in automatically as well.

Reply:
Ah, didn't see Jeremy's response in the other view, given that we had to pin it recently, I'm happy to pin it again now.  What was the reason for pinning it? Is there a ticket somewhere for unpinning it?

Reply:
For the `random` import I'm confused about the pinning, we never pinned it in py35.py which is what we were using in production and I don't see it in constraints.txt so did we actually pin it?

Reply:
Here's the PR: https://github.com/edx/edx-platform/pull/25788 -- it was against base.in, but might have been intended for Python 3.

Reply:
There isn't yet a newer release of random2; a community member pointed out in Slack that it was the only unpinned dependency in the file, so Ned preemptively pinned it to avoid problems later on.